### PR TITLE
added typehints

### PR DESCRIPTION
**Pull request by Tscunami**
Added typehints for each method
Also added docstring for init method inside Form class
Closing #18

### CODE REVIEW

File: urpaform/__init__.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
+""" __init__.py """
```
Comment (Slavaqq) [inline urpaform/__init__.py:1]: How it relates to type hints?

File: urpaform/elements.py [hunk 2]
```diff
@@ -38,15 +38,15 @@ class EditElement(_FormElement):
...
-        clear_keys=("CTRL+A", "DEL"),
-        default_value="",
-        send_method="writing",
-        paste_keys="CTRL+V",
-    ):
+        element: urpa.AppElement,
+        show_in_log: bool = True,
+        allow_check: bool = True,
+        value_is_in: str = "value",
+        clear_keys: tuple = ("CTRL+A", "DEL"),
```
Comment (Slavaqq) [inline urpaform/elements.py:45]: Use typing.Tuple

File: urpaform/form.py [hunk 3]
```diff
@@ -11,22 +14,32 @@
...
-    def __enter__(self):
+    def __enter__(self) -> Form:
         return self
 
-    def __exit__(self, exc_type, exc_value, exc_tb):
+    def __exit__(self, exc_type: Any, exc_value: Any, exc_tb: Any) -> None:
         self.complete()
 
-    def add(self, *args):
+    def add(self, *args: Union[Tuple[_FormElement, str], List[Tuple[_FormElement, str]]]) -> None:
```
Comment (Slavaqq) [inline urpaform/form.py:42]: List is too strict type.

**Review by Slavaqq — CHANGES_REQUESTED** [review]
Thank you. There are few things I want to change before I approve this pull requests.

**Comment by Tscunami** [comment]
Thank you for your feedback. Check out my last commit 1b14b0a , all required changes are fixed here

File: urpaform/__init__.py (hunk 1 above)
Comment (Tscunami) [reply urpaform/__init__.py]: It doesn't relate, I decided to remove this.

File: urpaform/form.py (hunk 3 above)
Comment (Tscunami) [reply urpaform/form.py]: I changed it from List to Iterable in the last commit.

File: urpaform/elements.py (hunk 2 above)
Comment (Tscunami) [reply urpaform/elements.py]: I changed it to Tuple[str, str]

**Review by Slavaqq — APPROVED** [review]
Thank you

**Pull request merged**